src/test/regress/expected/multi_explain.out

```diff
@@ -1115,3 +1116,85 @@ Custom Scan (Citus INSERT ... SELECT via coordinator)
         ->  Append
               ->  Function Scan on generate_series s
               ->  Function Scan on generate_series s_1
+-- explain with recursive planning
+EXPLAIN (COSTS OFF, VERBOSE true)
+WITH keys AS (
+  SELECT DISTINCT l_orderkey FROM lineitem_hash_part
+),
+series AS (
+  SELECT s FROM generate_series(1,10) s
+)
+SELECT * FROM series JOIN keys ON (s = l_orderkey) JOIN orders_hash_part ON (s = o_orderkey)
+ORDER BY s;
+Sort
+  Output: remote_scan.s, remote_scan.l_orderkey, remote_scan.o_orderkey, remote_scan.o_custkey, remote_scan.o_orderstatus, remote_scan.o_totalprice, remote_scan.o_orderdate, remote_scan.o_orderpriority, remote_scan.o_clerk, remote_scan.o_shippriority, remote_scan.o_comment
```

Comment:
Very minor, feel free to ignore: The output is pretty crowded. Making the query return 1 column could make the output more readable.